pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -440,6 +441,37 @@
   ai_resources_desc_03: "Supports integration of Artificial Intelligence knowledge throughout K-12. Offer free professional development, webinars, conference presentations, and peer-to-peer learning opportunities."
   ai_resources_btn_aria_label_03: "Explore the AI for Teachers website"
 
+  ai_pl_webinar_hero_heading: "AI 101 for Teachers"
+  ai_pl_webinar_hero_desc: "This is a comprehensive online learning series for any teacher and educator interested in the groundbreaking world of artificial intelligence (AI) and its transformative potential in education. Partners Code.org, Khan Academy, ISTE, ETS, and TeachAI are offering engaging sessions with renowned experts that will demystify AI, explore responsible implementation, address bias, and showcase how AI-powered learning can revolutionize student outcomes. Join us on this journey of exploration and empowerment, and unlock the future of teaching with and about AI."
+
```

Comment:
@dmantonyuk can you remove the empty line breaks so all of the `ai_pl_webinar` strings are grouped together? I've been using empty line breaks to differentiate between pages, and it's it's easier to see all the strings that belong to a single page when scrolling.

Reply:
done